packages/ckeditor5-restricted-editing/tests/manual/restrictedediting.js

```diff
@@ -90,4 +111,11 @@ async function reloadEditor( config ) {
 	}
 
 	window.editor = await ClassicEditor.create( document.querySelector( '#editor' ), config );
+	window.editor.plugins._plugins.delete( 'ImageBlockEditing' );
```

Comment:
This looks like a hack. You could, for example, adjust the list of loaded plugins and use [removePlugins](https://ckeditor.com/docs/ckeditor5/latest/api/module_core_editor_editorconfig-EditorConfig.html#member-removePlugins) instead.

Reply:
Yeah, its look a hack. But unfortunetely we can not use `removePlugins`, because `Image` requires `ImageBlockEditing`. `Image` is in the `ArticlePluginSet` then its difficult to exclude this plugin. So the simplest way is just remove `ImageBlockEditing` after building. Or we could just import every package from `ArticlePluginSet` except `Image`.